.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+version: 2
+jobs:
+  build:
+
+    docker:
+      - image: circleci/node:8
```

Comment:
worth using `8-alpine`?

Reply:
I'm using `circleci` docker images,
https://hub.docker.com/r/circleci/node/tags/

Unfortunately there's none image with Alpine. But this is nice question: Move foward with `8-alpine` (https://hub.docker.com/_/node/)?